R/decisiontreeutils.R

```diff
@@ -361,3 +398,34 @@ outputDTResultsAlteryx <- function(results, config) {
   title(main="Pruning Plot", line=5)
   invisible(dev.off())
 }
+
+#' process for converting to results list from config and data
+#'
+#' @param config list of configuration options
+#' @param data list of datastream objects
+#' @return list of results or results
+#' @export
+process <- function(config, data) {
+  # To get run-over-run consistency, set the seed
+  set.seed(1)
+
```

Comment:
I would think about moving this to the UI and letting the user set it. While I agree that it is good to have run-over-fun consistency, we should make it explicit rather than pushing it under the hood.